readme.md

```diff
@@ -33,72 +33,104 @@ To read further about the file generation features for each year, please refer t
 Clean files will pass the HMDA edits (business rules for data submission). These files are used as the base for generating files that will fail edits. Running the following scripts will create the edits_files directory and a data file that will pass the HMDA edit checks. The file will have a number of rows set in a YAML clean file configuration for each directory. Other variables, such as data ranges can also be set in the configuration files.
 
 For 2019:
-1. Run `cd 2019/python` to navigate to the python directory. 
-2. Adjust the [2019 Clean File Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/clean_file_config.yaml) for bank name, LEI, Tax ID and number of rows. 
-3. Run `python3 generate_2019_clean_files.py`
-4. The clean test file should appear in a new edits_files directory under `2019/edits_files/clean_files/{Bank Name}/` with the filename `clean_file_{Number or Rows}_{Bank Name}.txt`. 
+1. Navigate to the 2019/python directory.
+2. The default configuration for the test bank, filename, and row count contain the following values:
```

Comment:
Please preview your formatting to ensure that your newlines are rendering properly.

Reply:
Sure - Everything is being formatted as intended.